i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md

```diff
@@ -179,6 +206,15 @@ Release date: `October 26, 2022`
 
 ## Centreon MBI
 
+### 22.10.2
+
+Release date: `soon`
+
+#### Bug fixes
+
+- [Core] Improved database column size.
```

Comment:
Enhancement?

Reply:
only a technical one, that fixes potential issues related to this column in the database.